scipy/stats/tests/test_quantile.py

```diff
@@ -144,3 +178,14 @@ def test_size_0(self, axis, keepdims, xp):
             out_shape.pop(axis)
         res = stats.quantile(xp.zeros(tuple(shape)), 0.5, axis=axis, keepdims=keepdims)
         assert res.shape == tuple(out_shape)
+
+    @pytest.mark.parametrize('method',
+        ['inverted_cdf', 'averaged_inverted_cdf', 'closest_observation'])
+    def test_transition(self, method, xp):
+        # test that values of discontinuous estimators are correct when
+        # p*n + m - 1 is integral.
+        x = np.arange(8., dtype=np.float64)
+        p = np.arange(0, 1.125, 0.125)
```

Comment:
Oops.
```suggestion
        p = np.arange(0, 1.0625, 0.0625)
```
But let's let test run.